README.md

```diff
@@ -123,8 +103,5 @@ spec:
   mig_controller_image_fqin: quay.io/konveyor/mig-controller:release-1.2.2-hotfix-nopvs
   [...]
  ```
- 
-  3. Create a MigPlan and MigMigration covering the same namespaces migrated with `pvc-migrate`.
- 
 
-   
+  3. Create a MigPlan and MigMigration covering the same namespaces migrated with `pvc-migrate`.
```

Comment:
This is not you, my blind copy errors but can you 

s/pvc-migrate/imagestream-migrate

Reply:
Should be fixed.